wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/DotView.swift

```diff
@@ -24,7 +24,7 @@ final class DotView: UIView {
     private let circleView = ShapeView()
     private let centerView = ShapeView()
     private var userObserver: NSObjectProtocol!
-    private var clientsObserverTokens: [NSObjectProtocol] = []
+    private var clientsObserverTokens: [NSObjectProtocol?] = []
```

Comment:
why do we need to store nil token here, can we just check if the token exist before adding it?
or maybe I didn't get this change

Reply:
fixed